packages/next-app/src/app/internalmanager/[network]/[operationKind]/[tokenAddress]/page.tsx

```diff
@@ -115,6 +115,19 @@ export default function Page({
     return <Spinner />;
   }
 
+  if (chain?.name.toLowerCase() !== params.network) {
+    return (
+      <div className="w-full rounded-3xl items-center py-16 px-12 md:py-20 flex flex-col h-full">
+        <div className="text-center text-amber9 text-3xl">
+          You are on the wrong network
+        </div>
+        <div className="text-white text-xl">
+          Please change to {params.network}
```

Comment:
would be cool to have a small button here for the users to click and switch networks, will create an issue!

Reply:
https://linear.app/bleu-llc/issue/BAL-308/create-button-to-switch-networks